dealroom_firestore_connector/__init__.py

```diff
@@ -59,9 +60,24 @@ def get(doc_ref, *args, **kwargs):
             __log_exception(3, doc_ref, identifier, True)
             return ERROR
 
+def _update_last_edit(doc_ref):
+    """If the document reference points to the history collection
+    then update its "last_edit" field to the currrent datetime. This
+    datetame is parsed as a Timestamp in the document.
+
+    Args:
+        doc_ref: Firestore object reference to the document
+    """
+    _path = doc_ref.path.split("/", 1)
+    if len(_path) > 1 and _path[0] == "history":
+        doc_ref.update({"last_edit": datetime.now()})
```

Comment:
```suggestion
        # this will give the same time as `SERVER_TIMESTAMP` did
        from datetime import datetime, timezone
        utc_dt = datetime.now(timezone.utc)
        doc_ref.update({"last_edit": utc_dt})
```
the import is shown here but not intended to be committed here

Reply:
Solved in 5d92b5e